plexus-java/src/test/java/org/codehaus/plexus/languages/java/jpms/BinaryModuleInfoParserTest.java

```diff
@@ -66,17 +62,17 @@ public void testJarDescriptor() throws Exception {
     }
 
     @Test
-    public void testMultiReleaseJarDescriptor() throws Exception {
+    void testMultiReleaseJarDescriptor() throws Exception {
         JavaModuleDescriptor descriptor = parser.getModuleDescriptor(
                 Paths.get("src/test/resources/jar.mr.descriptor/jloadr-1.0-SNAPSHOT.jar"), JavaVersion.parse("17"));
 
         assertNotNull(descriptor);
         assertEquals("de.adito.jloadr", descriptor.name());
-        assertEquals(false, descriptor.isAutomatic());
+        assertFalse(descriptor.isAutomatic());
```

Comment:
as we use `org.hamcrest.MatcherAssert.assertThat` maybe better to use same pattern for consistency and have something such `assertThat(descriptor.isAutomatic(), is(true))` (the output in console reads better but I agree it's really subjective opinion :))

Reply:
I moved everything to assertj

Reply:
well really a matter of subjective taste I guess... :)